internal/commands/server/k8s_e2e_test.go

```diff
@@ -751,15 +751,14 @@ func TestTCPMeshService(t *testing.T) {
 			}, serviceOne.Name, "service not routable in allotted time")
 
 			// Force insecure cipher suite excluded from Consul API Gateway default
-			// cipher suites, but supported by Envoy defaults
-			//
-			// FIXME: why is this test not failing? attempting to connect with only
-			// this cipher suite should be rejected by the Consul API Gateway listener
-			// default config
+			// cipher suites, but supported by Envoy defaults, limit max version to
+			// TLS 1.2 to ensure cipher suite config is applicable. The error returned
+			// is a bit odd, but is expected for this failure case.
 			checkTCPTLSRoute(t, listenerOnePort, &tls.Config{
 				InsecureSkipVerify: true,
+				MaxVersion:         tls.VersionTLS12,
 				CipherSuites:       []uint16{tls.TLS_RSA_WITH_AES_128_CBC_SHA},
-			}, serviceOne.Name, "service not routable in allotted time")
+			}, "remote error: tls: unknown certificate authority", "service not routable in allotted time")
```

Comment:
can you change the expectation message here? it seems like your expectation is an error with this message, correct? probably should say something like `expected error not received` or something like that, may even want to do something like wrap the error in the helper function so we actually know what happened.

Reply:
Oops, so adding `t.Log` error logging clearly showed that I should've been expecting a different error message `"remote error: tls: handshake failure"`, which actually makes a lot more sense.